tests/update_test.m

```diff
@@ -0,0 +1,126 @@
+classdef Update_test < matlab.unittest.TestCase
+  %UPDATE_TEST contains unit tests for `git.update`
+  
+  properties
+    % A char array mock `FETCH_HEAD` file, so we can fetch code without
+    % manipulating repo's actual `FETCH_HEAD`.
+    FetchHeadFake
```

Comment:
use `file.modDate` mock fun instead of this

Reply:
This won't work, because what we would need to do is, within `git.update` actually modify the datenum corresponding to the last modified date of `FETCH_HEAD`. If you have another solution that you think is better than the current mock `FETCH_HEAD` file for `update_test`, let me know